packages/react-components/src/templates/TagsPageHeader.tsx

```diff
@@ -22,44 +23,71 @@ const textStyles = css({
 });
 
 const controlsStyles = css({
-  padding: `${networkContentTopPadding} ${contentSidePaddingWithNavigation(
+  padding: `${defaultContentTopPadding} ${contentSidePaddingWithNavigation(
     8,
   )} 0`,
 });
 
+const styles = css({
+  display: 'grid',
+  gridTemplateColumns: 'auto min-content',
+  gridColumnGap: rem(18),
+  alignItems: 'center',
+  justifySelf: 'center',
+});
+
 type TagsPageHeaderProps = {
   tags: string[];
-  setTags?: (tags: string[]) => void;
   loadTags?: ComponentProps<typeof MultiSelect>['loadOptions'];
-};
+  setTags: (tags: string[]) => void;
+} & Pick<
+  ComponentProps<typeof Filter<CRNTagSearchEntities>>,
+  'filters' | 'onChangeFilter' | 'filterOptions'
+>;
 
 const TagsPageHeader: React.FC<TagsPageHeaderProps> = ({
   tags,
-  setTags = noop,
+  setTags,
+  filters,
+  filterOptions,
+  onChangeFilter,
   loadTags = noop,
 }) => (
   <header>
     <div css={visualHeaderStyles}>
       <Display styleAsHeading={2}>Tags Search</Display>
       <div css={textStyles}>
```

Comment:
this `maxWidth` from `textStyles` is making it look a little bit weird in the desktop view

<img width="1431" alt="Screenshot 2023-11-28 at 09 23 07" src="https://github.com/yldio/asap-hub/assets/16595804/985ad556-9613-4a7b-962d-655d38f355ed">